syntax = "proto3";

package game_schemes;
import "game-state-schema.proto";
import "session-state-schema.proto";

message InitialStateResponseSchema{
  int32 playerId = 1;
  game_schemes.SessionStateSchema sessionState = 2;
  game_schemes.GameStateSchema gameState = 3;
}